FORMAT_JSON.md

```diff
@@ -0,0 +1,121 @@
+# Implementation Details: JSON
+
+## General Notes
+
+JSON is a format build on attribute-value pairs (dictionaries) and arrays.
+Its most common representation is as strings.
+
+Thus, in order to differentiate between openPMD **attributes** and **data sets** slight naming conventions need to be introduced.
+An example implementation is provided in [openPMD-api](https://github.com/openPMD/openPMD-api) and in doubt, its conventions shall apply.
+
+The following JSON keys (names) are reserved for openPMD usage with JSON:
+- `attributes`
+- `data`
+
+## Datatype Conventions
+
+The the attributes are wrapped into an `data` dictionary at the location where they would usually be stored.
+Inside the `data` dictionary, values are stored as an array.
+On the same level as the `data` dictionary, a `datatype` key-value pair is defined for identification of the string-serialized type.
```

Comment:
I think this explanation is a bit convoluted? Shouldn't we rather define clearly the representation of an openPMD dataset in JSON, i.e. something like:
> An openPMD dataset is represented by a JSON object with keys `attributes`, `data` and `datatype`: `{"attributes": <attributes>, "data": <data>, "datatype": <datatype>}`. In here, `<attributes>` is [put further explanations here].
An openPMD group is represented by a JSON object with key `attributes`, and additionally for each contained dataset or subgroup a key equivalent to the dataset's/subgroup's name.

Maybe also give a note that `data` is an n-dimensional array in row-major layout.

Reply:
Good idea, thank you, clarified now.